source/core/types/ut_test.tpb

```diff
@@ -54,6 +54,9 @@ create or replace type body ut_test as
 
     a_listener.fire_before_event(ut_utils.gc_test,self);
     self.start_time := current_timestamp;
+    
+    -- report to application_info
+    ut_utils.set_action(self.object_owner||'.'||self.object_name||'.'||self.name);
```

Comment:
can you move this call to `ut_executable`, so we have visibility into before/after procedures that get executed?